examples/counterfleetautoscaler.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# [Stage:Alpha]
+# [FeatureFlag:CountsAndLists]
+# Example of a FleetAutoscaler - this is used to scale a Fleet automatically depending on the
+# available capacity of a Counter across the fleet.
+#
+
+#
+# For a full reference and details: https://agones.dev/site/docs/reference/fleetautoscaler/
+#
+
+apiVersion: "autoscaling.agones.dev/v1"
+kind: FleetAutoscaler
+metadata:
+ name: fleet-autoscaler-counter
+spec:
+ fleetName: fleet-example
+ policy:
+   type: Counter # Counter based autoscaling
+   counter:
+     # The name of the Counter.
+     key: players
+     # Size of a buffer of counted items that are available in the Fleet.
```

Comment:
Is this size of a buffer based on the counter capacity that is available over the current aggregate counter capacity in the Fleet?

Reply:
We're overloading the word "Count" here, it really means "Available Capacity" which is Capacity - Count. Should we rename this something besides Count or Capacity? Perhaps minAvailability, maxAvailability, bufferAvailability (or something else)?

Reply:
Redid the naming to clarify it's min/maxCapacity = total Capacity (not available capacity as stated above).